tests/rules/pds3/test_coiss_xxxx.py

```
@@ -0,0 +1,326 @@
+        product_dict = test_pdsf.opus_products()
+        product_pdsfiles = []
+        for pdsf_lists in product_dict.values():
+            for pdsf_list in pdsf_lists:
+                for pdsf in pdsf_list:
+                    product_pdsfiles.append(pdsf)
+
+        # Filter out the metadata/documents products and format files
+        product_pdsfiles = []
+        for pdsf in product_pdsfiles:
+            if isinstance(pdsf, pds3file.Pds3File):
+                if pdsf.voltype_ != 'metadata/' and pdsf.voltype_ != 'documents/':
```

Ruff (0.15.21)

[warning] Consider merging multiple comparisons: `pdsf.voltype_ not in {'metadata/', 'documents/'}`.

Merge multiple comparisons

(PLR1714)